src/views/Collectibles/components/GlobalCheckClaimStatus.tsx

```diff
@@ -55,10 +60,16 @@ const GlobalCheckClaimStatus: React.FC<GlobalCheckClaimStatusProps> = ({ exclude
       setClaimableNfts(claimable)
     }
 
+    const fetchClaimAnniversaryStatus = async () => {
+      const canClaimAnniversary = await canClaim(account)
+      setCanClaimAnniversaryPoints(canClaimAnniversary)
+    }
+
     if (account) {
       fetchClaimStatus()
+      fetchClaimAnniversaryStatus()
     }
-  }, [account, canClaimBaller, canClaimLottie, canClaimLucky])
+  }, [account, canClaimBaller, canClaimLottie, canClaimLucky, canClaim])
```

Comment:
Also not too important but I think all of the claim conditions for lottery NFTs will now be falsy- so these can be removed. Nyan said that the other day but maybe double-check to be sure

Reply:
yep there is a time limit on the contract

Reply:
done here 6389cae8dfad153b5863930b92f2748adfaac6d9